docs/articles/templates/notes-warnings/requesthook-onresponse-no-stub-headers.md

```diff
@@ -0,0 +1,3 @@
+> Important! You can not change response body or its headers inside a `RequestHook.onResponse()` call. If you edit properties of `event.headers` or `event.body` inside `onResponse()`, TestCafe ignores these changes.
```

Comment:
The `RequestHook.onResponse` method takes a response from server. If you edit properties of `event.headers` or `event.body` inside `onResponse()` it affects the current request hook only. This modification will not affect the tested application response. So, the data in `onResponse` method of other request hooks is not changed.